variables.tf

```diff
@@ -121,12 +121,14 @@ variable "username" {
   description = "Username for the master DB user"
   type        = string
   default     = null
+  sensitive   = true
 }
 
 variable "password" {
   description = "Password for the master DB user. Note that this may show up in logs, and it will be stored in the state file"
   type        = string
-  default     = ""
+  default     = null
+  sensitive   = true
```

Comment:
`sensitive` was added in Terraform 0.14. I would not use this feature still in the module.

Reply:
ah right, mixing up my inputs and outputs. reverted from the variables section